### PR TITLE
product display card styled, reconfigured for bulma customization

### PR DESCRIPTION
**Pull request by sstampher**
Created simple non-detailed product view. This product view is not functional yet (buttons don't do anything, etc.) and style or info displayed can be further edited down the line.

I reconfigured a bunch of stuff in order to be able to customize Bulma. This included installing the node-sass package, removing the cdn and replacing it with a node-sass generated css file, and making some edits to package.json. If these changes that I made interfere with the back-end processes or anything else that is going on, let me know and I'll make the appropriate changes.

### CODE REVIEW

**Comment by justin-cook-developer** [comment]
I'm not familiar with node-sass, but if it produces a css file, I think the produced css file can be git ignored?

File: package.json [hunk 1]
```diff
@@ -7,10 +7,12 @@
...
   },
   "scripts": {
+    "css:build": "node-sass --omit-source-map-url ./public/style/mystyles.scss ./public/style/style.css",
+    "css:watch": "npm run css:build -- --watch",
     "test": "jest --config=jest.config.js -i --forceExit",
     "test:dev": "jest --config=jest.config.js -i --watchAll",
     "start": "npm run server",
-    "start:dev": "npm run build:dev & npm run server:dev",
+    "start:dev": "npm run css:watch & npm run build:dev & npm run server:dev",
     "build": "./node_modules/.bin/webpack  --config webpack.production.config.js",
```
Comment (justin-cook-developer) [inline package.json]: I think you might have to add `& npm run css:build` to the build script for the compiled scss file to be produced in production?

File: public/style/morestyles.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.center {
```
Comment (justin-cook-developer) [inline public/style/morestyles.css]: Is this file produced by scss and node-sass? If it is, I think it should be ignored.

Comment (sstampher) [reply public/style/morestyles.css]: No, this is an extra file that I am using to write custom css classes that I had been re-using quite a bit. If I make a small modification that I'll never use again, then I've been doing that in an in-line style tag on the component. If I find that I am making the same in-line style change in several different areas, then I make that into a class and write it here. So far, styling has been a combo of pure bulma, in-line css mods, custom classes like in here, and bulma.sass mods (mostly to change default colors, fonts).

Comment (sstampher) [reply public/style/morestyles.css]: The file produced by node-sass is style.css, the large one that probably won't load

File: package.json (hunk 1 above)
Comment (sstampher) [reply package.json]: yeah this is the stuff I was unsure about, I am sure you are correct. I was worried about these changes not showing up in production, but unsure which scripts to modify in order to make that happen. Thanks a lot I'll definitely add it

Comment (sstampher) [reply package.json]: The git-ignore issue I am honestly unsure about - this is my first time using a git-ignore file and also using node-sass, I will look into it but if you have any opinion on this I'm open to going with it

File: public/style/morestyles.css (hunk 2 above)
Comment (justin-cook-developer) [reply public/style/morestyles.css]: Got it! Thanks!

File: package.json (hunk 1 above)
Comment (justin-cook-developer) [reply package.json]: I am pretty sure that edit to the build script is required. So build would be `./node_modules/.bin/webpack  --config webpack.production.config.js & npm run css:build` where css:build is `path/to/binary/in/node_modules --omit-source-map-url ./public/style/mystyles.scss ./public/style/style.css`

Comment (justin-cook-developer) [reply package.json]: That way the scss will be compiled and produce the file.

Comment (justin-cook-developer) [reply package.json]: As for git ignoring, I'm pretty sure you should add `/public/style.css` to gitignore b/c it is the produced file, and we only need to track the file that produces it.

Comment (sstampher) [reply package.json]: yeah all of this makes sense, thanks a lot, i'll push up those changes in a few minutes

Comment (sstampher) [reply package.json]: Just to clarify, when adding the npm run css:build to the build script, should I change css:build to:

path/to/binary/in/node_modules --omit-source-map-url ./public/style/mystyles.scss ./public/style/style.css

I don't need to include anything about node-sass in the new css:build code?

Comment (sstampher) [reply package.json]: I am unable to push up any changes, github is giving me a really hard time and errors I've never seen before

**Pull request merged**